addons/mail/static/src/messaging/widget/discuss/discuss.js

```diff
@@ -51,23 +51,25 @@ const DiscussWidget = AbstractAction.extend({
         this.action = action;
         this.actionManager = parent;
         this.controlPanelParams.modelName = 'mail.message';
+        this.discuss = undefined;
```

Comment:
We already have `env`, and discuss is `env.messaging.discuss`, so I'm not sure if we need to store discuss too.

At best I would create a getter, based on `env`.

Alternative approach:

we only store `discuss`, and we access `env` with `discuss.env` and messaging with `discuss.messaging`